packages/core/nuxt-theme-module/theme/pages/Category.vue

```diff
@@ -440,6 +443,26 @@ export default {
       changeFilters(selectedFilters.value);
     };
 
+    const handleAddItemToCart = async (product, quantity) => {
```

Comment:
I'm wondering if it wouldn't be better to externalize all Notification objects to some other dir (like `assets/notifications`). We will have a lot of them and it will be easy to duplicate the code and make it less readable:

It would be better to have precofigured notifications in one object and have them parametrized with some context-specific information. WDYT?
 
for example

```js
sendNotification.cart.addItem.success(product._name)
sendNotification.cart.addItem.failure(error.value.addItem.message)
```

Then below code will look like this

```js
if (error.value.addItem) {
  sendNotification.cart.addItem.success(product._name)
} else {
  sendNotification.cart.addItem.failure(error.value.addItem.message)
}
```
@lukaszjedrasik @andrzejewsky @Fifciu WDYT?

Reply:
(check my comment below at first: https://github.com/vuestorefront/vue-storefront/pull/5425/files#r561064044) 

I like the idea of having an external file with notification handlers. Especially, when I see that we have duplicated code in the implementation of the first method.

### A few suggestions from me
- As we will have only `then` block (check comment below). Maybe it would be a good idea to have just one function called `handler` instead of `success` and `failure`.
- Let's send just a one argument - an object: `{productName: product._name, error: error.value.addItem}`

Reply:
@Fifciu so you're suggesting something like this?
```js
addToCart({ product, auantity}) = () => {
  addItem({product,quantity}).then(() => 
      sendNotification.cart.addItem({productName: product._name, error: error.value.addItem}
  )
}
```

Reply:
Yes exactly. Wdyt?